src/js/components/StarRating/StarRating.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Star, StarOutline } from 'grommet-icons';
+import { RadioButtonGroup } from '../RadioButtonGroup';
+
+export const StarRating = ({
+  fillColor,
+  name,
+  label,
+  scale = 5,
+  value,
+  onChange,
+  outlineColor,
+  ...rest
+}) => {
+  const [rating, setRating] = useState(value);
+
+  const options = [];
+  for (let i = 0; i < scale; i += 1) {
+    options.push(i);
+  }
+
+  return (
+    <RadioButtonGroup
+      direction="row"
+      options={options}
+      value={rating}
+      name="starRating"
```

Comment:
I don't think we have default `name` on other inputs, so thinking we should remove. But you might want to take a peak at how other inputs are using formContext:

TextInput.js for example
```
    const [value, setValue] = formContext.useFormInput({
      name: readOnly ? undefined : name,
      value: valueProp,
    });
```

Select
```
    const [value, setValue] = formContext.useFormInput({
      name,
      value: valueProp,
      initialValue: defaultValue || '',
    });
```

Reply:
aw okay yeah that makes sense but since its StarRating do we want to use that as a default?

since the other inputs are more flexible with how the user wants

Reply:
Took out `name` and since Im reusing RadioButtonGroup this works fine with putting `name` on formfield with default values